recipe/meta.yaml

```diff
@@ -1,20 +1,17 @@
 {% set name = "zodbpickle" %}
-{% set version = "1.0.4" %}
-{% set bundle = "tar.gz" %}
-{% set hash = "25d35f7c5ed95689d788b4da2a201990368b7abf032df0974632d0f8c5b39058" %}
+{% set version = "2.0.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ bundle }}
-  sha256: {{ hash }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: e0d9930f46d41f086a763f40a12d014c46bbc5e19e728c8d66d7a06fef616739
 
 build:
-  preserve_egg_dir: True
```

Comment:
Is this not needed anymore? Unless we are sure, i'd be better to keep it.

Reply:
I thought it is no longer needed when we install from pip.

Reply:
OK. IDK so let's go without it!